### ISSUE TITLE
Adversarial training losses do not match with those in the paper

### ISSUE THREAD

**Issue by ZijiaLewisLu** [text-machine-lab/adversarial_decomposition#5]
Thank you for releasing the code. 

I am reading "Adversarial Decomposition of Text Representation" paper and the code. In the paper, the discriminator is said to be trained with Wasserstein Loss instead of Cross Entropy. However, in the code it is trained using Cross Entropy Loss and a entropy loss not mentioned in the paper.  

I am wondering why there is the inconsistency between the code and the paper? Is the code obsolete or Wasserstein loss turns out to be not good?

Thanks.
